dstar/starclass/double_support.C

```diff
@@ -198,9 +198,11 @@ void put_state(double_state d,
  *  make_profile --
  *-----------------------------------------------------------------------------
  */
+/* Disabled because get_new_binary returns NULL, so this is broken
```

Comment:
Does this need/warrant a fix?

Reply:
No idea?

Reply:
If neither of us know, is there anyone else who can answer that?